### ISSUE TITLE
zoom and pan functionality for second y-axis is not working

### ISSUE THREAD

**Issue by amitgobare23** [chartjs/chartjs-plugin-zoom#127]
Hi team,

I successfully implement zoom and pan functionality for first y-axis(two y-axis in my Line chart). but now my requirement is i have to add zoom and pan functionality on second y-axis....please help me out asap....
Thanks in advance!

**Comment by patrislav1** [comment]
I just ran into the same issue.

Turns out that chartjs-plugin-zoom needs the [axis type (category, linear, logarithmic etc.)](http://www.chartjs.org/docs/latest/axes/cartesian/) set in order to work. And if the user doesn't set that axis type, chartjs sets it to linear by default, BUT only for the first y-axis.
(at least for some graph types - I ran into this when using a scatter graph)

I worked around that by setting the axis type explicitly for all axes.
`config.options.scales.yAxes[i].type = 'linear';`

Hope that helps
